src/js/pensions/components/IntroductionPage.jsx

```diff
@@ -20,18 +20,23 @@ class IntroductionPage extends React.Component {
         <FormTitle title="Apply for Veterans pension benefits"/>
         <p>Equal to VA Form 21P-527EZ (Application for Pension).</p>
         <SaveInProgressIntro
+          prefillEnabled={this.props.route.formConfig.prefillEnabled}
           pageList={this.props.route.pageList}
-          resumeOnly
+          startClass="no-text-transform"
+          startText="Start the Pension Application"
           {...this.props.saveInProgressActions}
-          {...this.props.saveInProgress}/>
+          {...this.props.saveInProgress}>
+          Please complete the 21-527EZ form to apply for pension benefits.
+        </SaveInProgressIntro>
+        <h4>Follow the steps below to apply for a Veterans pension.</h4>
         <div className="process schemaform-process schemaform-process-sip">
           <ol>
             <li className="process-step list-one">
               <div><h5>Prepare</h5></div>
               <div><h6>To fill out this application, you’ll need your:</h6></div>
               <ul>
                 <li>Social Security number or VA file number (required)</li>
-                <li>Military history (required)</li>
+                <li>Military history —— information like service dates, discharge character and rank that's commonly found on the DD214 or other separation documents (required)</li>
```

Comment:
I don't see this change mentioned in the issue. If it's just in the screenshot, I would want to confirm that it's an intentional change.

Reply:
Thanks @jbalboni, good point, could @peggygannon @stacyhlinn please let us know?

Reply:
@cehsu Thanks for checking. The line should be:

Military history (required) ....as it is on the current page.

Reply:
Thanks @peggygannon, this text has been reverted to current version.